src/components/organisms/Menu/style.scss

```diff
@@ -1,86 +1,145 @@
-.header__menu {
-    position: fixed;
-    top: 60px;
-    left: -100%;
-    width: 100%;
-    height: 100%;
-    background-color: var(--main-bg-color);
-    transition: left 0.5s;
-    overflow: auto;
-    z-index: 20;
-
-    &--active {
-        @extend .header__menu;
-        left: 0;
-    }
-
-    .header__list {
-        padding: 0 10px 10px;
-        text-transform: uppercase;
-        font-size: 12px;
-        letter-spacing: 1px;
-        font-family: var(--main-text-font);
-        cursor: pointer;
-        justify-self: end;
+.header {
+    &__menu {
+        position: fixed;
+        top: 60px;
+        left: -100%;
+        width: 100%;
+        background-color: var(--main-bg-color);
+        transition: left 0.5s;
+        overflow: auto;
+        z-index: 20;
     
-        li {
-            list-style: none;
-            margin: 0 0 10px;
-            border-top: 2px solid var(--main-border-color);  
-            padding: 7px 10px 0;
-            line-height: 35px;
-    
-            &:first-child {
-                border-top: 0;    
-            }
-
-            .header__link, .header__sublink {
-                text-decoration: none;
-                color: var(--main-color);
-            
-                &:hover {
-                    color: var(--brown-color);
-                }
-            }
+        &--active {
+            @extend .header__menu;
+            left: 0;
         }
-
-        .header__submenu {
-            position: fixed;
-            top: 60px;
-            left: 100%;
-            width: 100%;
-            padding: 0 10px 0 5px;
-            background-color: var(--main-bg-color);
-            transition: left 0.5s;
-            overflow: auto;
-            z-index: 20;
-        
-            &--active {
-                @extend .header__submenu;
-                left: 0;
-                padding-right: 10px;
-            }
+    
+        &__list {
+            padding: 0 10px 10px;
+            text-transform: uppercase;
+            font-size: 12px;
+            letter-spacing: 1px;
+            font-family: var(--main-text-font);
+            cursor: pointer;
+            justify-self: end;
         
             li {
```

Comment:
A safer approach is to style classes, not whole tag selectors.

Reply:
Changed.